apps/src/templates/teacherDashboard/EditSectionForm.jsx

```diff
@@ -205,6 +181,11 @@ class EditSectionForm extends Component {
     const showLoginTypeField =
       !isNewSection && changeableLoginTypes.includes(section.loginType);
 
+    // These are server-side experiments, which are passed down to the client
+    const showCodeReviewEnabledCheckbox =
```

Comment:
This should work for now but is not future-proof. I had trouble coming up with a good future-proof way of only showing this setting for CSA courses and couldn't come up with anything.

Reply:
I'm guessing we don't have access to the course here?

Reply:
Hmm we might actually. I'll look. Though I think we'll still have to hardcode the courses here but that is a bit better than the pilots.

Reply:
Are there course settings similar to the level settings? i.e. I think certain courses you can only view if you're over 13 and some are hidden by default, etc.

Reply:
I think the list of courses is populated by a server call. I just got distracted but I was trying to pull through the course name and check against that (we do have the course id here but I don't want to hard code in an id that will be environment-dependent).

Reply:
So thinking a bit more on this I realized that this dialog is also already kind of broken for pilots and it doesn't pre-fill the course if assigning a pilot course. Which means that accessing the course info seems like a pretty fragile approach. I'm going to keep this as-is for now but continue to think on this and I can always update later. Thanks for the ideas all!

Reply:
Oh sorry, replied before I saw Ben's comment! 

> re our CSA pilot scripts part of some family (not sure if this is the right word in cplat any more) that will be reused across each year's version of the course?

Yes! Currently one is in the `csa` family name (which I don't see changing) and the other in `csa-pilot-facilitator` (which likely won't be reused). I'm going to stick with this for now like I said but this thread has given me a lot of ideas that I can hopefully follow-up on this week.